numpy/random/_generator.pyx

```diff
@@ -3345,7 +3345,8 @@ cdef class Generator:
     def multivariate_normal(self, mean, cov, size=None, check_valid='warn',
                             tol=1e-8, *, method='svd'):
         """
-        multivariate_normal(mean, cov, size=None, check_valid='warn', tol=1e-8)
+        multivariate_normal(mean, cov, size=None, check_valid='warn',
+                            tol=1e-8, *, method='svd')
```

Comment:
Hmm. Does anyone know if it is OK to break a signature line like this? If not, we could make an exception to the 80 character limit.